d500px.info.yml

```diff
@@ -1,6 +1,8 @@
 name: 500px Integration
-type: module
```

Comment:
Not sure if this should be removed :)

according to D8 docs, this is needed.

> Ensure that the file has the line:
> type: module

https://www.drupal.org/docs/8/creating-custom-modules/let-drupal-8-know-about-your-module-with-an-infoyml-file